resources/static/js/Analytics.js

```diff
@@ -0,0 +1,409 @@
+        var ContentList;
+        var ContentMovements;
+        var OnsiteTempTotalOne = 0;
+        var OnsiteTempTotalTwo = 0;
+        var OnsiteTempTotalThree = 0;
+        var OnlineTempTotalOne = 0;
+        var OnlineTempTotalTwo = 0;
+        var OnlineTempTotalThree = 0;
+        
+       function LoadTotals(){  
+    	   for (var i = 0; i < ContentList.length; i++) {
+    		   switch (ContentList[i].seasonCode) {
+    		     case 1:
+    			   OnsiteTempTotalOne = OnsiteTempTotalOne + ContentList[i].onsiteSales;
+    			   OnlineTempTotalOne = OnlineTempTotalOne + ContentList[i].onlineSales;
+    			   break;
+    		     case 2:
+        		   OnsiteTempTotalTwo = OnsiteTempTotalTwo + ContentList[i].onsiteSales;
+        		   OnlineTempTotalTwo = OnlineTempTotalTwo + ContentList[i].onlineSales;
+        	       break;
+        	     case 3:
+        		   OnsiteTempTotalThree = OnsiteTempTotalThree + ContentList[i].onsiteSales;
+        		   OnlineTempTotalThree = OnlineTempTotalThree + ContentList[i].onlineSales;
+        	       break;
+        	       }
+    		   }
+    	   }
+
+        function LoadGraphics(){
+            var ctx = document.getElementById('myChart').getContext('2d');
+                    var myChart = new Chart(ctx, {
+                        type: 'line',               
+                        data: {
+                            labels: ['Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio','Julio'
+                            ,'Agosto','Septiembre','Octubre','Noviembre','Diciembre'],
+                            datasets: [{
+                                label: 'Importe de ventas de manera presencial',
+                                data: [ContentList[0].onsiteSales, ContentList[1].onsiteSales, ContentList[2].onsiteSales,
+                                    ContentList[3].onsiteSales, ContentList[4].onsiteSales, ContentList[5].onsiteSales,
+                                    ContentList[6].onsiteSales, ContentList[7].onsiteSales, ContentList[8].onsiteSales,
+                                    ContentList[9].onsiteSales, ContentList[10].onsiteSales, ContentList[11].onsiteSales],
```

Comment:
Deberían definirse constantes para los subíndices para entender que es cada cosa. Alternativamente, siendo que los subíndices parecen ser consecutivos, se podría armar esta nueva lista con un for en una variable aparte y luego ser suministrada a `data`.